ui/src/components/admin/Notifications/PublishNotifications.js

```diff
@@ -0,0 +1,324 @@
+import React, { useContext } from 'react';
+import { NotificationsContext } from './NotificationsController';
+import NOTIFICATION_TYPES from './NotificationTypes';
+import {
+  acknowledgePublishStatus,
+  checkPublishStatus,
+} from 'components/admin/Model/actions/Publish';
+import { PUBLISH_ERRORS, PUBLISH_TYPES } from 'utils/constants';
+
+export const PublishNotificationsContext = React.createContext([{}, () => {}]);
+
+export const PublishNotificationsProvider = ({ children }) => {
```

Comment:
Modelled after [ui/src/components/admin/Notifications/GenomicVariantImportNotifications.js](https://github.com/nci-hcmi-catalog/portal/blob/develop/ui/src/components/admin/Notifications/GenomicVariantImportNotifications.js), but with all the GDC-related code removed, import error code removed, and converted to a Context so that `ModelsManagerController` can use it.

Reply:
Lots of duplicated/similar code between the linked file above and here where the only change is `import` -> `publish`.

A refactor could probably save a lot of duplication, but wasn't in the cards here due to time constraints. Possibility for a future ticket though.